examples/wallet-transfer/Cargo.toml

```
@@ -0,0 +1,13 @@
+[package]
+name = "fuels-example-wallet-transfer"
+publish = false
+version = "0.1.0"
+authors = ["Fuel Labs <contact@fuel.sh>"]
+edition = "2021"
+homepage = "https://fuel.network/"
+license = "Apache-2.0"
+repository = "https://github.com/FuelLabs/fuels-rs"
+
+[dependencies]
+fuels-rs = { version = "0.9.1", path = "../../packages/fuels-rs" }
+tokio = { version = "1.10.1", features = ["full"] }
```

Prefer major.minor instead of major.minor.patch for non-Fuel deps.
```suggestion
tokio = { version = "1.10", features = ["full"] }
```